generators/server/templates/quarkus/src/test/resources/application.properties.ejs

```diff
@@ -59,10 +63,25 @@ quarkus.swagger-ui.path=/swagger-ui
 quarkus.native.additional-build-args=\
     -H:ResourceConfigurationFiles=resources-config.json,\
     --initialize-at-run-time=<%= packageName %>.security.RandomUtil
-
 <%if (buildTool === 'gradle') { _%>
+
 # Docker image build properties, for more information https://quarkus.io/guides/container-image
 quarkus.container-image.group=
 quarkus.container-image.name=<%= baseName %>
 quarkus.container-image.tag=latest
 <%_ } _%>
+
+<%if (authenticationType === 'oauth2') { _%>
+# Oauth 2.0 and OIDC
+quarkus.oidc.enabled=true
+quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/jhipster-test/
+quarkus.oidc.client-id=web_app
+quarkus.oidc.credentials.secret=web_app
+quarkus.oidc.application-type=service
+quarkus.oidc.authentication.scopes=profile,address,email,jhipster,microprofile-jwt,offline_access,phone
+quarkus.oidc.authentication.cookie-path=/
+quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
+quarkus.oidc.authentication.restore-path-after-redirect=false
+
+jhipster.oidc.logout-url=http://localhost:8180/auth/realms/jhipster-test/protocol/openid-connect/logout
```

Comment:
It'd be nice if these values were fake and not tied to a particular IdP (identity provider).

Reply:
Done